composer.json

```diff
@@ -14,11 +14,11 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.*",
-        "satooshi/php-coveralls": "0.*"
+        "phpunit/phpunit": "^4.0|^5.0",
```

Comment:
As you noted, the problem is that PHPUnit5 is available on PHP 5.6+ and it will break the tests on lower versions.
PHP 5.5 will be unmaintained by the end of June. PHP 5.3 and 5.4 are already unsupported. It makes sense to get rid of the tests that use these versions.